templates/overrides/navigation/oe-corporate-blocks-eu-footer.html.twig

```diff
@@ -1,4 +1,14 @@
 {% extends "@oe_whitelabel/overrides/navigation/oe-corporate-blocks-neutral-footer.html.twig" %}
+
+{% block accessibility_link %}
+  {% set _accessibility_link = pattern('link', {
```

Comment:
I am not a fan of using the link pattern, it doesn't support things like url attributes that might be in the url object.
We might leave it for now though.

Reply:
we resolved that by patching with our bcl_link twig function